pages/index.tsx

```diff
@@ -55,17 +68,28 @@ export const getServerSideProps: GetServerSideProps = async ({ req, locale }) =>
   const logger = isProd ? pino({}) : pino({ prettyPrint: true })
   logger.info(req)
 
-  // Make the API request and return the data.
-  const claimData = await queryApiGateway(req)
+  let errorCode: number | null = null
+  let claimData: Claim | null = null
+  let scenarioContent: ScenarioContent | null = null
 
-  // Run business logic to get content for the current scenario.
-  const scenarioContent = getScenarioContent(claimData)
+  try {
+    // Make the API request and return the data.
+    claimData = await queryApiGateway(req)
```

Comment:
Nit: Any reason not to declare `claimData` here instead of the outer scope?

Reply:
Good catch! Pushed a commit to correct this.